frontend/src/components/dashboard/DashboardTransactions.jsx

```diff
@@ -8,87 +8,124 @@ import { Row, Col } from "react-bootstrap";
 
 import Content from "../utils/Content";
 import TransactionsList from "../transactions/TransactionsList";
+import TransactionsApi from "../../libraries/explorer-wamp/transactions";
+import FlipMove from "react-flip-move";
 
-export default ({ transactions }) => (
-  <Content
-    title={<h2>Recent Transactions</h2>}
-    icon={<TransactionIcon style={{ width: "22px" }} />}
-    size="medium"
-    border={false}
-    className="dashboard-transactions"
-  >
-    <Row className="px-0">
-      <Col xs="1" className="pr-0">
-        <div className="dashboard-blocks-hr-parent">
-          <div className="dashboard-blocks-hr" />
-        </div>
-      </Col>
-      <Col xs="11" className="px-0 dashboard-transactions-list">
-        <TransactionsList
-          transactions={transactions}
-          viewMode="compact"
-          reversed
-        />
-        <Row noGutters>
-          <Col xs="1" className="dashboard-transactions-icon-col" />
-          <Col xs="6">
-            <Link href="transactions">
-              <a className="dashboard-footer">View All</a>
-            </Link>
+export default class extends React.Component {
+  state = {
+    transactions: this.props.transactions.slice(0, 10)
```

Comment:
If you need less info, query for less info, do no slice it on the frontend. I used 15 transactions as default as we don't have a pagination just yet, and 10 was not enough.